Client/Extensions/PaymentRequestExtension.swift

```diff
@@ -23,6 +23,8 @@ class PaymentRequestExtension: NSObject {
     fileprivate enum PaymentRequestErrors: String {
         case NotSupportedError = "NotSupportedError"
         case AbortError = "AbortError"
+        case TypeError = "TypeError"
+        case RangeError = "RangeError"
```

Comment:
Can we camelCase these please?

I.e. `NotSupportedError` → `notSupportedError`